src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,19 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main() {
 
+        Machine truck = new Truck();
+        truck.doWork();
+        truck.stopWork();
+
+        Machine excavator = new Excavator();
+        excavator.doWork();
+        excavator.stopWork();
+
+        Machine bulldozer = new Bulldozer();
+        bulldozer.doWork();
+        bulldozer.stopWork();
+
```

Comment:
let's create a `Machine` array. After that you can go through this array, using for loop and call methods `doWork()` and `doStop()` in it. And remove empty lines at the start and at the end of the method)